README.md

```diff
@@ -183,6 +183,82 @@ Blank nodes are not supported.
   }
 }
 ```
+#### Resource Object Fragmentation Shape
+
+A fragmentation strategy that groups triples by (subject) resources.
+It generates shapes and shape index files in the root of the iri container for each information type defined in `shapeDirectory`. 
+The `shapeDirectory` must contain a `config.json` following the exemple below.
+The `shape` must be a path relative to the `shapeDirectory` and the `directory` must be the name of container of the subject.
+The shape located at `shape` must be of the format [`ShExC`](https://shex.io/shex-semantics/index.html#shexc).
+
+`./shape/config.json`
+```json
+{
+    "shapes": {
+        "comments": {
+            "shape": "comments.shexc",
+            "directory": "comments"
+        },
+    }
+}
+```
+
+An exemple of the `component.js` configuration is presented below.
+A probability for the generation of shape information of eachs resource can be defined with `generationProbability`.
+
+```json
+{
+  "fragmentationStrategy": { 
+        "@type": "FragmentationStrategyShape",
+        "shapeDirectory": "./shape",
+        "tripleShapeTreeLocator": true,
+        "generationProbability": 0.5
```

Comment:
Maybe this probability is something we can move to another (wrapper-based) strategy, so we can also use if for other fragmentation strategies. What do you think?

Reply:
I agree